mlflow/tracking/fluent.py

```diff
@@ -72,25 +72,29 @@
 
 
 class DefaultSetContextVar:
-    def __init__(self, contextvar, default: Callable):
-        self.contextvar = contextvar
+    def __init__(self, name: str, default: Callable):
```

Comment:
/ai This is Python code. `default` is a function that takes no arguments and returns `Any`. Is it possible to improve the type hint?